backend/httpclient/max_bytes_reader.go

```diff
@@ -24,12 +24,13 @@ var ErrResponseBodyTooLarge = errors.New("http: response body too large")
 // MaxBytesReader prevents clients from accidentally or maliciously
 // sending a large request and wasting server resources.
 func MaxBytesReader(r io.ReadCloser, n int64) io.ReadCloser {
-	return &maxBytesReader{r: r, n: n}
+	return &maxBytesReader{r: r, n: n, a: n}
 }
 
 type maxBytesReader struct {
 	r   io.ReadCloser // underlying reader
 	n   int64         // max bytes remaining
+	a   int64         // the actual limit
```

Comment:
Nit. Naming could be improved for these fields :)

Reply:
@marefr An attempt has been made here https://github.com/grafana/grafana-plugin-sdk-go/pull/1072/commits/6441e0b45c3abd52b40dd6793eb078107bf62713 but I am not sure if those are good enough.